aospy/test/test_calc_basic.py

```diff
@@ -143,5 +146,55 @@ def setUp(self):
             'dtype_out_vert': 'vert_int'
         }
 
+class TestCalcAttrs(unittest.TestCase):
```

Comment:
Sorry if I didn't mention this before, but we are transitioning away from the class-based, `unittest.Testcase` style of tests in favor of `py.test`'s more function-based tests.  The latter requires less boilerplate and provides additional functionality.

So all you have to do here is delete this `TestCalcAttrs` class, and turn it's `test_units_description_attrs` method into a module-level function.

When you do that, I think it will flow better if you place it after your helper functions, rather than before.